install.sh

```diff
@@ -521,6 +536,90 @@ do_install() {
 			echo_docker_as_nonroot
 			exit 0
 			;;
+			
+		sles)
+			arch=$(uname -m)
+			if [[ $arch != "s390x" ]]; then
+				echo "Docker-CE is not supported on SLES for $arch!!!"
+				exit 1
+			else
+				echo "Installing Docker-CE on $arch!"
+			fi
```

Comment:
Same here (`[[ ... ]]` is also a bashism / non-POSIX);

```suggestion
			if [ "$(uname -m)" != "s390x" ]; then
				echo "Docker-CE is currently only supported SLES for s390x"
				exit 1
			fi
```

Reply:
Updated the script with mentioned changes.